### DISCUSSION TITLE
Install in a folder (not in the root) like https://www.my-website.com/INSTALL_BLOG_HERE/posts/test10/index.html

### DISCUSSION THREAD

**Discussion by Artchibald** [colbyfayock/next-wordpress-starter#518] [General]
I changed the homepage url in package.json, ran "npx next build" and "npx next export" but all the js links and css links were broken when I placed it on the server in that folder via ftp. Thanks in advance for your help.

This is where I need it:
https://www.my-website.com/INSTALL_BLOG_HERE/posts/test10/index.html

**Comment by colbyfayock** [comment]
hey @Artchibald this project doesn't have any control over that aspect. please refer to the Next.js docs

this might help: https://nextjs.org/docs/pages/api-reference/next-config-js/basePath
